ui/packages/catalog-ui-search/src/main/webapp/react-component/inputs/enum-input/enum-input.js

```diff
@@ -53,23 +105,49 @@ const EnumInput = ({
           onChange={setInput}
         />
       </TextWrapper>
-      <Menu value={value} onChange={onChange}>
+      <Menu value={value} onChange={onChange} class="fa">
         {allowCustom &&
           displayInput && (
             <EnumMenuItem value={input}>{input} (custom)</EnumMenuItem>
           )}
         {filteredSuggestions.map(suggestion => {
           return (
-            <EnumMenuItem key={suggestion.value} value={suggestion.value}>
+            <EnumMenuItem
+              title={
+                isAttributeUnsupportedHelper(settingsModel, suggestion)
+                  ? 'Attribute is unsupported by the content store(s) selected'
```

Comment:
In DDF, content stores are called sources. We'll have to fetch this string using the i18n service